### DISCUSSION TITLE
Homepage downloads blank “index” file instead of loading site on NextJS 13

### DISCUSSION THREAD

**Discussion by forntoh** [vercel/next.js#42834] [Help] [answered]
### NextJS 13

Whenever I open my [website](https://forntoh.dev), it requests to download a file instead of displaying the page.

On safari mobile, the message is "Do you want to download forntoh.dev".

How do I prevent that?

Hosted on Vercel.

![image](https://user-images.githubusercontent.com/27886627/201482405-5e6c7634-0577-41aa-9f52-3d339dd3621e.png)

**Comment by forntoh** [comment, marked as answer]
Fixed in v13.0.3

**Reply by akoskm** [reply]
It's possible that I was experiencing the same in https://github.com/vercel/next.js/discussions/43028.

I'm bumping the version to ^13.0.4 and see how it goes.

**Reply by akoskm** [reply]
I can confirm that the download is not happening after upgrading to v13.0.3.
